GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
I've been reviewing this idea, and wanted to report that besides integration with csproj's assembly and NuProj's package version number, the integration this PR currently benefits from with NB.GV includes:

* The nuspec in the package includes the git commit ID in two places.
* NB.GV's generated code files don't trigger StyleCop.Analyzers to complain the way GitVersion's evidently do. GitVersion's files lack the 'code generated' file headers that turn stylecop.analyzers off.